src/ralph/dc_view/urls/api.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.conf.urls import patterns, url
+
+from ralph.dc_view.views.api import (
+    DCAssetsView,
+    DCRacksAPIView
+)
+
+
+urlpatterns = patterns(
```

Comment:
See
https://docs.djangoproject.com/en/1.8/topics/http/urls/#syntax-of-the-urlpatterns-variable and https://docs.djangoproject.com/en/1.8/releases/1.8/#django-conf-urls-patterns - from Django 1.8 `patterns` is deprecated.